server/src/models/Gallery/Image.model.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable consistent-return */
+const mongoose = require('mongoose')
+
+const { Schema } = mongoose
+
+const ImageSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+    index: { unique: true },
+    lowercase: true
+  },
+  url: {
+    type: String,
+    required: true,
+    index: { unique: true }
+  },
+  collections: [{
+    type: Schema.Types.ObjectId,
+    ref: 'Collection'
+  }],
```

Comment:
I'm not use if I understand what is collection used for, we should probably take a call about it.